index.html

```diff
@@ -50,6 +50,141 @@ <h1>I’m Eugene. <br>
                                <li><a href=""><img src="images/portfolio5.svg" alt=""></a></li>                       
                            </ul>
             </div>
-
         </div>
-    </main>
\ No newline at end of file
+            <div class="work-section">
+                <div class="work-container">
+                    <div class="card-container">
+                      <div class="card"><img src="images/Snapshoot Portfolio (1).svg" alt=""></div>  
+                           <div class="left-block">
+                            <h2>Tonic</h2>
+                            <ul class="position">
+                                <li>CANOPY <i class="fa fa-circle"></i></li>
+                                <li>Back End Dev  <i class="fa fa-circle"></i></li>
+                                <li>2015</li>
+                            </ul>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                            <ul class="programming-lang">
+                                <li><a href="#">HTML</a></li>
```

Comment:
- [ ] Kindly fix the list of project technologies.
- [ ] Your Design
<img width="519" alt="Screenshot 2023-07-14 at 16 04 44" src="https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81444338/26ff86f6-8323-4a10-91ca-2b94b142270b">

- [ ] Figma Design
<img width="354" alt="Screenshot 2023-07-14 at 16 11 31" src="https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81444338/ebc33b95-fa41-48d3-b73c-f9cffac8d043">